src/openrct2/localisation/StringIds.h

```diff
@@ -2758,8 +2758,7 @@ enum
     STR_THEMES_OPTION_RCT1_RIDE_CONTROLS = 5282,
     STR_THEMES_OPTION_RCT1_PARK_CONTROLS = 5283,
     STR_THEMES_OPTION_RCT1_SCENARIO_SELECTION_FONT = 5284,
-    STR_CHEAT_EXPLODE = 5285,
-    STR_CHEAT_EXPLODE_TIP = 5286,
+
```

Comment:
Do we want this new line?

Reply:
I put it in because there is a gap, and also because the strings following are from a different area than the ones preceding.